site/data/dynamo.ts

```diff
@@ -615,12 +637,17 @@ export const removeOrganisation = async (orgId: string) => {
     );
 };
 
-export const removeSocialMediaPostFromDisruption = async (index: number, disruptionId: string, id: string) => {
+export const removeSocialMediaPostFromDisruption = async (
+    index: number,
+    disruptionId: string,
+    id: string,
+    isTemplate?: boolean,
+) => {
     logger.info(`Removing socialMediaPost ${index} in disruption (${disruptionId}) in DynamoDB table...`);
 
     await ddbDocClient.send(
         new DeleteCommand({
-            TableName: disruptionsTableName,
+            TableName: isTemplate ? templateDisruptionsTableName : disruptionsTableName,
             Key: {
                 PK: id,
                 SK: `${disruptionId}#SOCIALMEDIAPOST#${index}`,
```

Comment:
At the moment the "remove post" button doesn't work on the template flow because the SK has #EDIT on the end for templates.

Will need to conditionally create the SK based on isTemplate.  Something like:
```
const getSK = isTemplate
        ? `${disruptionId}#SOCIALMEDIAPOST#${index}#EDIT`
        : `${disruptionId}#SOCIALMEDIAPOST#${index}`;
```

Then pass in ```SK: `${getSK}` ```

Reply:
done